tests/unit/CorrectNameRequest.spec.ts

```diff
@@ -76,6 +76,39 @@ describe('CorrectNameRequest', () => {
     await flushPromises()
 
     expect(wrapper.vm.isFormValid).toBe(true)
+    expect(wrapper.vm.nameRequestNumber).toEqual('NR 1234567')
+  })
+
+  it('verifies inputs when valid', async () => {
+    const wrapper = wrapperFactory()
+
+    // Verify Invalid before input
+    expect(wrapper.vm.isFormValid).toBe(false)
+
+    wrapper.vm.nameRequestNumber = 'NR 1234567'
+    wrapper.vm.applicantPhone = '123 456 7890'
+    wrapper.vm.applicantEmail = 'mock@example.com'
+
+    await flushPromises()
+
+    expect(wrapper.vm.isFormValid).toBe(true)
+    expect(wrapper.vm.nameRequestNumber).toEqual('NR 1234567')
+  })
+
+  it('verifies inputs when valid', async () => {
+    const wrapper = wrapperFactory()
+
+    // Verify Invalid before input
+    expect(wrapper.vm.isFormValid).toBe(false)
+
+    wrapper.vm.nameRequestNumber = '1234567'
+    wrapper.vm.applicantPhone = '123 456 7890'
+    wrapper.vm.applicantEmail = 'mock@example.com'
+
+    await flushPromises()
+
+    expect(wrapper.vm.isFormValid).toBe(true)
+    expect(wrapper.vm.nameRequestNumber).toEqual('NR 1234567')
```

Comment:
Thanks for the nice tests.

Can you please add a test(s) for leading and/or trailing spaces and extra spaces between "NR" and the number? Thanks.

Reply:
If there are leading or trailing spaces, the NR is invalid so far. Should the spaces be ignored and pass the validation?

Reply:
Yes, `  ` is invalid.

However, `  NR  1234567  ` should be  valid.

Reply:
@severinbeauvais the test cases have added. Please review.